spec/lib/index.spec.js

```diff
@@ -0,0 +1,434 @@
+"use strict";
+
+describe("lib/index.js", () => {
+    var morePromises, timer;
+
+
+    /**
+     * Create a delayed promise that resolves or rejects
+     *
+     * @param {number} ms
+     * @param {*} result
+     * @param {boolean} [isFailure=false]
```

Comment:
Doesn't look like isFailure actually defaults to `false`. Granted, `undefined` and `false` are treated the same way, but technically, there is a difference.